src/test/java/uk/gov/companieshouse/chd/order/api/service/OrderServiceTest.java

```diff
@@ -2,18 +2,14 @@
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.core.Is.is;
-import static org.junit.Assert.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
We have been avoiding wildcard imports :) Can we replace with implicit imports

Reply:
Done, imported individual package